charts/nr-k8s-otel-collector/examples/k8s/rendered/daemonset-configmap.yaml

```diff
@@ -211,6 +212,8 @@ data:
             to: resource["k8s.pod.uid"]
 
     processors:
+      
```

Comment:
are these spaces on purpose 👀

Reply:
The Render workflow added these changes. 
I'm not in control over anything in the examples/k8s/rendered directory. 
These are auto generated.